tasks/debian.yaml

```diff
@@ -1,16 +1,10 @@
 ---
-# - name: system packages | add nginx apt key
-#   become: true
-#   ansible.builtin.apt_key:
-#     keyring: /usr/share/keyrings/nginx-archive-keyring.gpg
-#     url:  https://nginx.org/keys/nginx_signing.key
-#     id: ABF5BD827BD9BF62
-
-
-# - name: system packages | Add nginx stable repository into sources list
-#   ansible.builtin.apt_repository:
-#     repo: "deb [arch=amd64,arm64 signed-by=/usr/share/keyrings/nginx-archive-keyring.gpg] http://nginx.org/packages/ubuntu jammy nginx"
-#     state: present
+- name: system packages | remove nginx
+  become: true
+  ansible.builtin.apt:
+    name: nginx
+    state: absent
+  when: nginx_remove
```

Comment:
What is the rationale behind adding this task?

Reply:
This was added in https://github.com/ome/ansible-role-nginx/pull/16/files. 

We had a problem to upgrade nginx, even when this role was demanding nginx 1.26, the resulting nginx was always 1.20.  This was because the default sources list was not listing the 1.26. The only solution was to remove nginx first (note that this is optional in the role), there is a new variable ``nginx_remove`` and rerun the role. Thus the removal of nginx was included as an option into the role.

Reply:
@khaledk2  I have added an explanation in https://github.com/ome/ansible-role-nginx/pull/17/commits/0284c8dfb935a490a7222c7dc872edd028965374